apps/projection/src/app/ui/card/card.component.scss

```diff
@@ -0,0 +1,11 @@
+.bg-light-green {
+  background-color: rgba(0, 250, 0, 0.1);
+}
+
+.bg-light-red {
+  background-color: rgba(250, 0, 0, 0.1);
+}
+
+.bg-light-blue {
+  background-color: rgba(0, 0, 250, 0.1);
+}
```

Comment:
that's a way of doing it. You can use css variable also. 
But for this example, you could only apply this class to the host, but applying class instead of an input.